docs/dev/developer.rst

```diff
@@ -26,18 +131,24 @@ Submit a `Pull Request <https://github.com/radis/radis/pulls>`__ from GitHub.
 Online tests will be run automatically. They will check for:
 
 - Physics test cases, to ensure that the code is still working as expected (see :ref:`Test Section <label_dev_test>` to run them locally).
-- Code format (see :ref:`Code linting below <label_linting>`).
+- Code format (see :ref:`Code Style and Linting <label_linting>`).
+- Dependency synchronization between ``pyproject.toml`` and ``environment.yml``
 
 .. include:: _test.rst
 
-.. include:: _linting.rst
-
 .. include:: _architecture.rst
 
 Help
 ----
 
-If you encounter any problem, please `report an Issue <https://github.com/radis/radis/issues?utf8=%E2%9C%93&q=is%3Aissue>`_ on GitHub.
+If you encounter any problem:
+
+1. Check if it's a known issue in our `GitHub Issues <https://github.com/radis/radis/issues>`__
+2. If not, please `open a new issue <https://github.com/radis/radis/issues/new/choose>`__ with:
+   - A quick summary
+   - Steps to reproduce
+   - Expected vs actual behavior
+   - Any relevant notes or attempted solutions
```

Comment:
Indent not displayed correctly on https://radis--734.org.readthedocs.build/en/734/dev/developer.html